build.sbt

```diff
@@ -36,7 +36,8 @@ lazy val commonSettings = Seq(
   scalacOptions ++= Seq("-target:jvm-1.8"),
   javacOptions ++= Seq("-source", "1.8"),
   // -target cannot be passed as a parameter to javadoc. See https://github.com/sbt/sbt/issues/355
-  Compile / compile / javacOptions ++= Seq("-target", "1.8")
+  Compile / compile / javacOptions ++= Seq("-target", "1.8"),
+  coverageEnabled := true
```

Comment:
what does this enable? collecting the metrics? how much extra time do the tests take if code coverage is enabled?

Reply:
This is to enable scoverage to collect testing metrics. You can either enable it in `build.sbt` by setting `coverageEnabled` to `true`, or by running `coverage` before running `test` within the sbt shell session; the full command becomes `sbt clean coverage test coverageAggregate`.

Reply:
since I dont want to enable the coverage by default (hence i asked to add the cmdline arg --coverage), we should not set this to true here right?

Reply:
Oupsi, I just corrected it. Thanks!